.github/workflows/plugin-stat.yml

```
@@ -0,0 +1,28 @@
+name: Pull plugin stats
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  pull-stats:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+        env:
+          gh_token: ${{ secrets.GH_TOKEN }}
+        run: |
+          node update-stats.js
+  commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Commit updates
+        run: |
+          git config --local user.name 'Obsidian Bot'
+          git config --local user.email 'admin@obsidian.md'
+          git commit -am "Update plugin stats"
```

```suggestion
          git commit -am "chore: Update plugin stats"
```

Just a suggestion to make it a semantic commit message.